src/tree/registries/auth/AzureOAuthHelper.ts

```diff
@@ -0,0 +1,35 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See LICENSE.md in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as request from 'request';
+import { ISubscriptionContext } from 'vscode-azureextensionui';
+import { acquireAcrAccessToken, acquireAcrRefreshToken } from '../../../utils/azureUtils';
+import { ICachedRegistryProvider } from '../ICachedRegistryProvider';
+import { IDockerCliCredentials } from '../RegistryTreeItemBase';
+import { IAuthProvider, IOAuthContext } from './IAuthProvider';
+
+export interface IAzureOAuthContext extends IOAuthContext {
+    subscriptionContext: ISubscriptionContext
+}
+
+class AzureOAuthHelper implements IAuthProvider {

```

Comment:
```suggestion
class AzureOAuthProvider implements IAuthProvider {

```
😬

Reply:
Heheh, that should have been obvious given the change to IAuthProvider 😆